packages/examples/source/src/index.ts

```diff
@@ -2,9 +2,5 @@ import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute, endpointSelector } from './adapter'
 import { makeConfig, NAME } from './config'
```

Comment:
Can we include the types here and include them in the export below?

```suggestion
import { makeConfig, NAME } from './config'
import * as types from './endpoint'
```